docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -615,4 +615,26 @@ SELECT * FROM input PARTITION BY col0 EMIT CHANGES;
 ### `ALL`, `WINDOWSTART` and `WINDOWEND` are now reserved identifiers
 
 Any query using these identifiers will need to be changed to either use some other identifier, or
-to quote them.
\ No newline at end of file
+to quote them.
+
+
+
+### Upgrading from confluentksqlDB 5.5 to confluentksqlDB 6
+
+In-place upgrades are supported from confluentksqlDB 5.5 to confluentksqlDB 6.
+
+#### Steps of "In-Place" upgrade :-
+
+1. Download the latest confluent version.
+2. Set it up with all required configs. In our case we mostly copied all configs from the current version(which is running) to the new version.
+3. Take Backup and then copy all UDF JAR to new version .(OPTIONAL STEP In case you don't have any UDF).
+4. Stop ksqlDB process of current version(5.5 in our case) and start newer version(6 in our case) with latest KsqlDB config file which you created in step2.
+5. KsqlDB will start executing queries and rebuild all states using command topic because the value of "ksql.service.id" is the same.
+6. Now KsqlDB Server with the latest version is up and running.
+
+#### Common Issues :-
```

Comment:
How about we rename this section "Troubleshooting an Upgrade"?

Reply:
Understood and changes done in my branch.